src/Propel/Generator/Model/Column.php

```diff
@@ -1739,6 +1739,7 @@ public static function generatePhpSingularName(string $phpName): string
     public function isContent(string $content): bool
     {
         $contentAttribute = $this->getAttribute('content');
+        // @todo: expression below has same operands - fix or kick?
 
         return $contentAttribute && strtoupper($contentAttribute) === strtoupper($contentAttribute);
```

Comment:
definitly doesnt make sense to have this nonsense operation/comparison :)

Reply:
That's for sure ... but not sure if this is just a typo and someone wanted to compare $content with $contentAttribute?